src/main/openapi/ds-image-analysis-openapi_v1.yaml

```diff
@@ -3,7 +3,22 @@ info:
   version: 'v1'
   title: '${project.name}'
   description: |
-    Image Analysis provides analytical tools for our image collections at KB. 
+    Image Analysis provides analytical tools for our image collections at KB. The endpoints of this API are grouped by categories. The primary category is color analysis.
+
+    **Color Analysis**
+    
+    The endpoints in this category all relates to working with colors of images. In this context it is usefull to have an understanding of some color theory. 
```

Comment:
I like that you define what we mean by colors in this context.

Typo: `useful`

Maybe provide a link to OKLab and SMK, where they are mentioned in the text? Also, it would be great if there was a link to the SMK palette. I don't know if it is online somewhere?

It's not "most dominant", just "dominant" - the wording is also used elsewhere in the API desciption.

Reply:
The SMK is  only available in their colorsearch here: but a link to that would also be nice: https://open.smk.dk/art/advanced/colors?q=*&page=0